lib/routes/settings/pos_settings_page.dart

```diff
@@ -159,6 +171,119 @@ class PosSettingsPageState extends State<_PosSettingsPage> {
     return widgets;
   }
 
+  Widget _buildExportItemsTile(PosCatalogBloc posCatalogBloc) {
+    return ListTile(
+      title: Container(
+        child: AutoSizeText(
+          "Items List",
+          style: TextStyle(color: Colors.white),
+          maxLines: 1,
+          minFontSize: MinFontSize(context).minFontSize,
+          stepGranularity: 0.1,
+          group: _autoSizeGroup,
+        ),
+      ),
+      trailing: Padding(
+        padding: const EdgeInsets.only(right: 0.0),
+        child: PopupMenuButton(
+          color: Theme.of(context).backgroundColor,
+          icon: Icon(
+            Icons.more_horiz,
+            color: Theme.of(context).iconTheme.color,
+          ),
+          padding: EdgeInsets.zero,
+          offset: Offset(12, 36),
+          onSelected: _select,
+          itemBuilder: (context) => [
+            PopupMenuItem(
+              height: 36,
+              value: Choice(() => _importItems(context, posCatalogBloc)),
+              child: Text('Import from CSV',
+                  style: Theme.of(context).textTheme.button),
+            ),
+            PopupMenuItem(
+              height: 36,
+              value: Choice(() => _exportItems(context, posCatalogBloc)),
+              child: Text('Export to CSV',
+                  style: Theme.of(context).textTheme.button),
+            ),
+          ],
+        ),
+      ),
+      onTap: () => _onChangeAdminPasswordSelected(),
+    );
+  }
+
+  void _select(Choice choice) {
+    choice.function();
+  }
+
+  Future _importItems(
+      BuildContext context, PosCatalogBloc posCatalogBloc) async {
+    return promptAreYouSure(
+            context,
+            "Import Items",
+            Text(
+                "Importing this list will override the existing one. Are you sure you want to continue?",
+                style: Theme.of(context).dialogTheme.contentTextStyle),
+            contentPadding: EdgeInsets.fromLTRB(24.0, 16.0, 24.0, 8.0),
+            cancelText: "NO",
+            okText: "YES")
+        .then((acknowledged) async {
+      if (acknowledged) {
+        File importFile = await FilePicker.getFile(
+            type: FileType.custom, allowedExtensions: ['csv']);
+
+        String fileExtension = path.extension(importFile.path);
+        if (fileExtension == ".csv") {
+          var action = ImportItems(importFile);
+          posCatalogBloc.actionsSink.add(action);
+          var loaderRoute = createLoaderRoute(context);
+          Navigator.of(context).push(loaderRoute);
+          action.future.then((_) {
+            Navigator.of(context).removeRoute(loaderRoute);
+            Navigator.of(context).pop();
+            showFlushbar(context, message: "Items were successfully imported.");
+          }).catchError((err) {
+            Navigator.of(context).removeRoute(loaderRoute);
+            var errorMessage = "Failed to import POS items.";
+            switch (err.toString()) {
+              case "INVALID_FILE":
```

Comment:
Better to have these strings as constants on the action. Or even better we can use customize errors as constants so we compare errors and not strings.

Reply:
[Done](https://github.com/breez/breezmobile/pull/357/commits/a662057afe38100bb96e5b5456ca5716ef2b4abb)